documentation/analytics/technical-overview.md

```diff
@@ -0,0 +1,73 @@
+# Technical Overview
+
+## Key technologies
+
+The analytics package is written in Python, analyzes and visualizes data using pandas and plotly, and exposes a command line interface using typer:
+
+- [pandas][pandas-docs] ([source code][pandas-source]) - Used to manipulate and analyze data
+- [plotly][plotly-docs] ([source code][plotly-source]) - Used to visualize the results of analysis
+- [typer][typer-docs] ([source code][typer-source]) - Used to build the CLI for the analytics package
+- [dynaconf][dynaconf-docs] ([source code][dynaconf-source]) - Used to manage configuration variables and secrets
+
+## Integrations
+
+In order to export data for analysis and then share the results of that analysis, the analytics package integrates with multiple third-party tools. These integrations are managed in the `src/analytics/etl` sub-directory, with each integration managed by its own module.
```

Comment:
`src/analytics/etl` should be replaced with `src/analytics/integrations` to reflect new directory structure.

Reply:
Thanks for catching this! Update it in this commit: [fix: References to analytics.etl package in docs](https://github.com/HHS/simpler-grants-gov/pull/984/commits/68896c86a5f97443a3fdc3ff553247696b709fe6)